Ambiguous-Squids/Bangular
# added cart partial

# Description
Added partial for cart, routing for partial, and controller for cart

## Steps to Test

1.  $ git fetch
2. $ git checkout nc-partials
3. $ http-server
4. Click on Cart on nav bar
5. Make sure you can see cart view (no info is in cart partial other than title right now)
